### ISSUE TITLE
Bug: extra time step writes out at end of netcdf file

### ISSUE THREAD

**Issue by lizaclark** [UW-Hydro/RVIC#85]
The netcdf history file produced by `rvic convolution` contains a dummy time step at the end. For example, in my file it is `"0000-01-01":
<pre>data:                                                                
time = "1980-01-01", "1980-01-02", "1980-01-03", "1980-01-04", "1980-01-05",  ...  "1980-12-23 12", "1980-12-24 12", "1980-12-25 12",
    "1980-12-26 12", "1980-12-27 12", "1980-12-28 12", "1980-12-29 12",
    "1980-12-30 12", "1980-12-31 12", "0001-01-01" ;
}</pre>

**Comment by lizaclark** [comment]
Note issue #70 might fix this.

**Comment by jhamman** [comment]
@lizaclark - can you show us the `[history]` section in your configuration file?

**Comment by lizaclark** [comment]
I should note that I just copied this from other users and haven't played much with the settings.
```
[HISTORY]

#--Number of History tapes.
RVICHIST_NTAPES = 1

#--Per tape series maximum number of time samples.
RVICHIST_MFILT  : 1000000

#--Per tape series history file density (i.e. output precision)
#    1=double precision
#    2=single precision
RVICHIST_NDENS  : 2

#--Per tape series history write frequency.
#  positive means in time steps
#  0 = monthly
#  negative means hours
RVICHIST_NHTFRQ : 1

#--Per tape series history write frequency.
#  A - Average, over the output interval.
#  I - Instantaneous, output the value at the output interval.
#  X - Maximum, over the output interval.
#  M - Minimum, over the output interval.
RVICHIST_AVGFLAG = A

#-- Per tape series history file format (char)
# Valid Values: grid, array
RVICHIST_OUTTYPE: grid
RVICHIST_NCFORM: NETCDF4

#-- Output parameter file compression options --#
# Descriptions of these options can be found in
RVICHIST_NETCDF_ZLIB: False
RVICHIST_NETCDF_COMPLEVEL: 4
RVICHIST_NETCDF_SIGFIGS: None

#-- Per tape series output units (char)
# Valid Values: kg m-2 s-1, m3/s
RVICHIST_UNITS: m3/s
</pre>

**Comment by jhamman** [comment]
closed via #86
